MeetupPlatformApi/Controllers/AuthenticationController.cs

```diff
@@ -46,34 +46,99 @@ public async Task<IActionResult> RegisterUser([FromBody] UserRegistrationDto reg
         var user = mapper.Map<UserEntity>(registrationDto);
 
         user.Password = BCrypt.HashPassword(user.Password);
+        await context.Users.AddAsync(user);
 
-        context.Users.Add(user);
+        var tokenPair = authenticationManager.IssueTokenPair(user);
+
+        await context.RefreshTokens.AddAsync(tokenPair.RefreshToken);
         await context.SaveChangesAsync();
 
-        var accessToken = authenticationManager.IssueAccessToken(user);
         var userInfoDto = mapper.Map<UserOutputDto>(user);
         var outputDto = new UserRegistrationResultDto
         {
             UserInfo = userInfoDto,
-            AccessToken = accessToken
+            TokenPair = new() { AccessToken = tokenPair.AccessToken, RefreshTokenId = tokenPair.RefreshToken.Id }
         };
         return CreatedAtAction(nameof(GetUserById), new {id = outputDto.UserInfo.Id}, outputDto);
     }
 
+    [HttpPost("refreshTokens/{id:guid}")]
+    public async Task<IActionResult> TokenRefresh(Guid id)
```

Comment:
Please, pass the whole `RefreshToken` and not only it's `Id`. Also, You should do it through `HTTP Body` and not `URL Path`.